src/components/card-edit.js

```diff
@@ -0,0 +1,178 @@
+const createFormEdit = () =>
```

Comment:
Предлагаю использовать другое наименование: `имяФайла` + `Component`

```suggestion
const cardEditComponent = () =>
```